src/test/scala/org/camunda/feel/impl/builtin/BuiltinListFunctionsTest.scala

```diff
@@ -403,4 +403,11 @@ class BuiltinListFunctionsTest extends AnyFlatSpec with Matchers with FeelIntegr
     eval(""" string join(["foo", 123, "bar"]) """) should be(ValNull)
   }
 
+  "A list is empty() function" should "return if the list is or not empty" in {
+
+    eval(" is empty([]) ") should be(ValBoolean(true))
+    eval(" is empty([ ]) ") should be(ValBoolean(true))
+    eval(" is empty([1]) ") should be(ValBoolean(false))
+    eval(" is empty([1,2,3]) ") should be(ValBoolean(false))
```

Comment:
:wrench: We should add one more test case with named parameters to ensure that the named parameter doesn't change.

```
eval(" is empty(list: [1]) ") should be(ValBoolean(false))
```